core/os/device/remotessh/commands.go

```diff
@@ -274,3 +275,61 @@ func (b binding) GetEnv(ctx context.Context) (*shell.Env, error) {
 	}
 	return e, nil
 }
+
+// ListFiles returns the executables in a particular directory as given by path
+func (b binding) ListExecutables(ctx context.Context, inPath string) ([]string, error) {
+	if inPath == "" {
+		inPath = "."
+	}
+	files, err := b.Shell("find", "\""+inPath+"\"", "-mindepth", "1", "-maxdepth", "1", "-type", "f", "-executable", "-printf", "%f\\\\n").Call(ctx)
```

Comment:
Use backticks for more readable strings `` `"` ``, or better yet - use `text.Quote()` ?

Reply:
I had tried text.Quote() there was an edge-case where it wasn't quoting. (I think it was being correct based on how it tries to work) but I cannot remember the specifics.